cypress/e2e/po/pages/global-settings/performance.po.ts

```diff
@@ -27,13 +36,44 @@ export class PerformancePagePo extends RootClusterPage {
     return new ModalWithCardPo();
   }
 
+  websocketCheckbox(): CheckboxInputPo {
+    return CheckboxInputPo.byLabel(this.self(), 'Disable websocket notifications');
+  }
+
+  incrementalLoadingCheckbox(): CheckboxInputPo {
+    return CheckboxInputPo.byLabel(this.self(), 'Enable incremental loading');
+  }
+
+  manualRefreshCheckbox(): CheckboxInputPo {
+    return CheckboxInputPo.byLabel(this.self(), 'Enable manual refresh of data for lists');
+  }
+
+  garbageCollectionCheckbox(): CheckboxInputPo {
+    return CheckboxInputPo.byLabel(this.self(), 'Enable Garbage Collection');
+  }
+
+  namespaceFilteringCheckbox(): CheckboxInputPo {
+    return CheckboxInputPo.byLabel(this.self(), 'Enable Required Namespace / Project Filtering');
+  }
+
+  websocketWebWorkerCheckbox(): CheckboxInputPo {
+    return CheckboxInputPo.byLabel(this.self(), 'Enable Advanced Websocket Web Worker ');
+  }
+
   applyButton() {
     return new AsyncButtonPo('[data-testid="performance__save-btn"]', this.self());
   }
 
-  restoresSettings() {
+  applyAndWait(value?: string): Cypress.Chainable {
+    cy.intercept('PUT', 'ui-performance').as(`apply${ value }`);
+    this.applyButton().click();
+
+    return cy.wait(`@apply${ value }`);
```

Comment:
this one too, for a 200

Reply:
same as above comment https://github.com/rancher/dashboard/pull/9458#discussion_r1460150799